docs/source/llm/getting-started.md

```diff
@@ -318,24 +457,24 @@ m = convert_pt2e(m, fold_quantize=False)
 DuplicateDynamicQuantChainPass()(m)
 
 traced_model = export(m, example_inputs)
-
 ```
 
-Additionally, add or update the to_backend() call to use XnnpackDynamicallyQuantizedPartitioner. This will instruct the lowering logic to emit the correct quantized operators.
+Additionally, add or update the `to_backend()` call to use `XnnpackPartitioner`. This instructs ExecuTorch to
+optimize the model for CPU execution via the XNNPACK backend.
 
-```
+```python
 from executorch.backends.xnnpack.partition.xnnpack_partitioner import (
-    XnnpackDynamicallyQuantizedPartitioner,
+    XnnpackPartitioner,
 )
+```
 
+```python
 edge_manager = to_edge(traced_model, compile_config=edge_config)
-
-# Lower to XNNPACK using the appropriate quantized partitioner.
-edge_manager = edge_manager.to_backend(XnnpackDynamicallyQuantizedPartitioner())
-
+edge_manager = edge_manager.to_backend(XnnpackPartitioner()) # Lower to XNNPACK.
 et_program = edge_manager.to_executorch()
 ```
-Finally, update the CMakeLists.txt to link the XNNPACK backend with the runner.
+
+Finally, ensure that the runner links against the `xnnpack_backend` target in CMakeLists.txt.
 
 ```
 add_executable(nanogpt_runner nanogpt_runner.cpp)
```

Comment:
is it main.cpp?

Reply:
Thanks. I updated the text here to be more clear.